common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -324,7 +324,7 @@ public BrokerSilentTokenCommandParameters brokerSilentParametersFromBundle(
                 .authenticationScheme(getAuthenticationScheme(context, brokerRequest))
                 .androidApplicationContext(context)
                 .accountManagerAccount(account)
-                .sdkType(brokerRequest.getSdkType())
+                .sdkType(brokerRequest.getSdkType() == null ? SdkType.MSAL : brokerRequest.getSdkType())
```

Comment:
add a test?

Reply:
this was caught in the test pass

Reply:
Right - we should add a unit test for this behavior of MsalBrokerRequestAdapter so that we don't require a manual test in order to catch it next time.

Reply:
Agreed. @tanmaymanolkar1 I suggest you create a work item in your backlog to add a unit test for this. I don't think we necessarily have to do it right now as I don't want to block the release process.

Reply:
That works.  Typically my ideal flow would be - write test exposing break, fix code to close break, break is now proven fixed.